terraform/cloudfoundry/elb_access_log_bucket.tf

```diff
@@ -0,0 +1,27 @@
+variable "elb_access_log_bucket_name" {
+#  default     = "elb-access-log"
+  default     = "elb-access-log-cf-droplets"
+}
```

Comment:
Why not?

I am bypassing the policy. We are allowed to create `*-cf-droplets` among others. That should be deleted once the PR on the polices is merged.